server/src/main/java/org/elasticsearch/common/geo/GeoJson.java

```diff
@@ -382,7 +382,7 @@ public static String getGeoJsonName(Geometry geometry) {
         return geometry.visit(new GeometryVisitor<>() {
             @Override
             public String visit(Circle circle) {
-                return "circle";
+                return "CIRCLE";
```

Comment:
this is for GeoJSON, not WKT. right?

these should all be capitalized using upper-camel-case/pascal-case

`Circle`, `MultiLineString`...

Reply:
Indeed! Got carried away :)